src/lib/config/defaultConfig.js

```diff
@@ -649,6 +649,11 @@ export const defaultConfig = {
     },
   },
   PATRONS: {
+    // this is what we get when we scan patron's badge-barcode
```

Comment:
this is very cern specific, I would say to name it something like `patronUniqueID` field?